### DISCUSSION TITLE
What am I doing wrong? unrecognized arguments: --use-directML

### DISCUSSION THREAD

**Discussion by Nimtronic** [lshqqytiger/stable-diffusion-webui-directml#397] [Q&A] [answered]
Since the last update, this is happening erverytime:
1. I create a new folder 
2. I open a console in the folder
3. git clone https://github.com/lshqqytiger/stable-diffusion-webui-directml.git
4. start \stable-diffusion-webui-directml\webui-user.bat
5. RuntimeError: Torch is not able to use GPU; add --skip-torch-cuda-test to COMMANDLINE_ARGS variable to disable this check
6. I had never received this error before. Add the argument in the webui-user.bat  
7. start \stable-diffusion-webui-directml\webui-user.bat
8. Start successful. start txt2img. CPU is used instead of the 7900XTX
9. close the session and add --use-directML as an argument
10.  launch.py: error: unrecognized arguments: --use-directML

I assume that I did not clone the fork from lshqqytiger but from Automatic1111. But how can that be after I use the correct URL in step 3? Can someone please explain this to me?

PS: before the update, everything worked without any problems. I have never had to add any extra arguments.

**Comment by lshqqytiger** [comment, marked as answer]
It is `--use-directml` not `--use-directML`.

**Reply by Nimtronic** [reply]
Thank you very much. It works now.
